### PR TITLE
Add transactions endpoint and fixes

### PR DESCRIPTION
**Pull request by dosvath**
## Problem

* No transactions endpoint
* Typos
* Missing descriptions

## Solution

* Add transactions endpoint
* Fix typos and add clarifications

## Testing Done

- [x] Spec builds using `redoc-cli serve`
- [x] Previewed changes locally

Screen Shot 2022-01-12 at 9 04 28 PM

### CODE REVIEW

File: openapi/openapi.yml [hunk 1]
```diff
@@ -296,7 +303,9 @@ paths:
       - Funding
       operationId: fundinInfo
```
Comment (ncassata) [inline openapi/openapi.yml:304]: Does the `operationId` value have a typo? Should the value be `fundingInfo`?

File: openapi/openapi.yml [hunk 2]
```diff
@@ -296,7 +303,9 @@ paths:
...
       summary: Information for PSE
-      description: Fetch the destination account info for initiating a PSE transfer.
+      description: Fetch the destination account info for initiating a PSE transfer. The information from this endpoint
+        can be used to initiate a PSE if the wallet has a bak account, or display the PSE instructions to the user if the wallet
```
Comment (ncassata) [inline openapi/openapi.yml:307]: Please correct the typo in the word `bak`.

File: openapi/openapi.yml [hunk 3]
```diff
@@ -449,14 +458,74 @@ paths:
           $ref: '#/components/responses/NotAllowed'
         '500':
           $ref: '#/components/responses/InternalError'
+  /account/transactions:
+    get:
+      tags:
+        - Monitoring
+      operationId: accountTransactions
+      summary: Account Transactions
+      description: Get a historical overview of a user's deposit/withdrawal transactions over the last 6 months.
```
Comment (ncassata) [inline openapi/openapi.yml:467]: For the MVP are we planning on only having the transactions over the last six months available? What are your thoughts about allowing the user to request the transactions for a specified time (e.g. all transactions over the past two weeks)?

**Review by ncassata — COMMENTED** [review]
Overall, looks good. Just a few comments.

File: openapi/openapi.yml (hunk 3 above)
Comment (dosvath) [reply openapi/openapi.yml:467]: We could do either endpoint: `6 months` or `specified (max 6 months)` , or both. Lmk what you think.

Comment (dosvath) [reply openapi/openapi.yml:467]: Btw, A&V limits us to a max of 6 months for both of their endpoints.

**Comment by dosvath** [comment]
Thanks @ncassata updated 👍

**Review by ncassata — APPROVED** [review]
LGTM! Great work

**Pull request merged**